### PR TITLE
Fixing Integration tests for 0.7.0

### PR DESCRIPTION
**Pull request by ygy**
- io.brooklyn.camp.brooklyn.JavaWebAppsIntegrationTest.testWithDbDeploy()
- io.brooklyn.camp.brooklyn.JavaWebAppsIntegrationTest.testWithPolicyDeploy()

### CODE REVIEW

File: usage/camp/src/test/resources/java-web-app-simple.yaml [hunk 1]
```diff
@@ -21,8 +21,8 @@ description: Single JBoss using Brooklyn
 origin: https://github.com/apache/incubator-brooklyn
 location: localhost
 services:
-- serviceType: brooklyn.entity.webapp.jboss.JBoss7Server
-  name: jboss1
+- serviceType: brooklyn.entity.webapp.tomcat.TomcatServer
```
Comment (mikezaccardo) [inline usage/camp/src/test/resources/java-web-app-simple.yaml]: `Tomcat8ServerImpl` is an option here if you'd like to use a newer version.

**Comment by mikezaccardo** [comment]
LGTM

**Comment by ygy** [comment]
@mikezaccardo it is using ```Tomcat8Server``` now

**Comment by neykov** [comment]
Good to merge.

**Pull request merged**